proto/provlabs/vault/v1/events.proto

```
@@ -360,3 +360,19 @@ message EventVaultFeeCollected {
   // outstanding_amount is the amount of AUM fee that remains unpaid after this collection (e.g., "20nhash").
   string outstanding_amount = 6;
 }
+
+// EventParamsUpdated is an event emitted when the module parameters are updated.
+message EventParamsUpdated {
+  // params are the newly updated module parameters.
+  Params params = 1;
```

Buf (1.66.1)

[error] field provlabs.vault.v1.EventParamsUpdated.params: unknown type Params

(COMPILE)